File: CS/MvcDashboard_ServerSideApi/Controllers/HomeController.cs
using System.Web.Mvc;

namespace MVCDashboard_ServerSideAPI.Controllers {
    public class HomeController : Controller {
        public ActionResult Index() {
            return View();
        }

        public ActionResult SalesDashboards() {
            return View("SalesView");
        }
        public ActionResult MarketingDashboards() {
            return View("MarketingView");
        }
    }
}

File: VB/MvcDashboard_ServerSideApi/Controllers/HomeController.vb
Imports System.Web.Mvc

Namespace MVCDashboard_ServerSideAPI.Controllers
    Public Class HomeController
        Inherits Controller

        Public Function Index() As ActionResult
            Return View()
        End Function

        Public Function SalesDashboards() As ActionResult
            Return View("SalesView")
        End Function
        Public Function MarketingDashboards() As ActionResult
            Return View("MarketingView")
        End Function
    End Class
End Namespace